widget/editor.go

```diff
@@ -259,75 +368,99 @@ func (e *Editor) processKey(gtx layout.Context) {
 			e.caret.scroll = true
 			e.scroller.Stop()
 			e.append(ke.Text)
+		// Complete a paste event, initiated by Shortcut-V in Editor.command().
 		case clipboard.Event:
 			e.caret.scroll = true
 			e.scroller.Stop()
 			e.append(ke.Text)
 		}
 		if e.rr.Changed() {
+			e.ClearSelection()
```

Comment:
If e.rr.Changed, why didn't the changer also update (clear) the selection?

Reply:
Good point.  Will remove.